tests/foreman/cli/test_errata.py

```diff
@@ -1551,3 +1552,60 @@ def test_install_applicable_package_to_registerd_host(chost):
     )
     assert len(applicable_packages) == 1
     assert REAL_RHEL7_0_2_PACKAGE_NAME in applicable_packages[0]['filename']
+
+
+@pytest.mark.tier2
+def test_downgrade_package_using_default_content_view_shows_errata(
```

Comment:
The test name is quite long. Can you try shortening it?

Reply:
Thank you for the review. Changes made. is it OK now?